cmd/check_container.go

```diff
@@ -168,17 +170,40 @@ func getContainerPolicyExceptions(ctx context.Context, pc pyxisClient) (policy.P
 
 func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 	if len(args) != 1 {
-		return fmt.Errorf("a container image positional argument is required")
+		return fmt.Errorf("a container image positional argument is required or a flag is not specified as --flag=value")
```

Comment:
This is a strange user-facing message to send. Why would a flag not being specified as `--flag=value` cause us to throw an error if `len(args) != 1`?

Reply:
I think I changed that when I tried on of the other settings. This should be removed.